packages/zowe-explorer/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 
 - The "Zowe Resources" panel is now hidden by default until Zowe Explorer reveals it to display a table or other data. [#3113](https://github.com/zowe/zowe-explorer-vscode/issues/3113)
 - Fixed behavior of logout action when token is defined in both base profile and parent profile. [#3076](https://github.com/zowe/zowe-explorer-vscode/issues/3076)
+- Fixed bug where old profiles remained in the Zowe Explorer tree views after deleting a Team configuration file. [#3124](https://github.com/zowe/zowe-explorer-vscode/issues/3124)
```

Comment:
Could we rephrase just a bit: Fixed bug that displayed obsolete profiles in the Zowe Explorer tree views after the associated team configuration file was deleted.

Reply:
Thanks, I like this phrasing better - addressed in 9dc2039